data/FilterList.json

```diff
@@ -16968,7 +16968,8 @@
     "licenseId": 5,
     "name": "Synology's own",
     "syntaxId": 2,
-    "viewUrl": "https://in0de.synology.me/own.txt"
+    "viewUrl": "https://in0de.synology.me/own.txt",
+    "isDeleted": true,
```

Comment:
```suggestion
    "isDeleted": true
```

Reply:
I think it's the trailing comma at the end of the json object. Debugged by looking at the fairly new "ling json" step [here](https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=4051&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=dd349f25-7095-58cb-1ea1-12e8abf37d4a) and validating that the json spec doesn't allow trailing commas [here](https://stackoverflow.com/a/201856/2343739).

Reply:
Well spotted. I hope I've taken care of it now.